test/msw-api/setup-server/use.test.ts

```diff
@@ -72,3 +72,24 @@ test('returns a mocked response from a one-time request handler override only up
   expect(anotherBookResponse.status).toBe(200)
   expect(anotherBookBody).toEqual({ title: 'Original title' })
 })
+
+test('returns a mocked response from a one-time request handler override only upon first request match with parallel requests', async () => {
+  server.use(
+    rest.get('https://test.mswjs.io/book/:bookId', (req, res, ctx) => {
+      return res.once(ctx.json({ title: 'One-time override' }))
```

Comment:
What if we add the `bookId` in the title, if we do this it's clear which request is caught here?

Reply:
I added a question about that, please have a look.

Reply:
I think this got answered, right?

Reply:
Indeed! I'll give it a go, thanks @timdeschryver